manifests/s/SublimeHQ/Sublimetext2/2.0.2/SublimeHQ.Sublimetext2.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SublimeHQ.Sublimetext2

```

```suggestion
PackageIdentifier: SublimeHQ.SublimeText.2
```
You'll also have to update the folder structure using GitHub's desktop client.